sprocket/util/tests/test_hdf5.py

```diff
@@ -28,9 +29,14 @@ def test_HDF5(self):
         tmp1d = tmph5.read(ext='1d')
         tmp2d = tmph5.read(ext='2d')
         tmph5.close()
-
         assert np.allclose(tmp1d, data1d)
         assert np.allclose(tmp2d, data2d)
 
+        # read/write and replace test
+        h5 = HDF5(path, 'a')
+        tmp1d = h5.read(ext='1d')
+        h5.save(data1d2, '1d')
+        h5.close()
```

Comment:
I suppose some `assert` should be added.

Reply:
Thanks. I add them.